src/NETCDF/dump_netcdf.h

```diff
@@ -40,12 +40,26 @@ class DumpNetCDF : public DumpCustom {
   virtual void write();
 
  private:
+   // type of quantity for per-atom values (used to get the unit)
+   enum Quantity {
+      Unknown = 0,
```

Comment:
Please make the enumerator members upper case to indicate that those are constants.

Reply:
sure! do you prefer me adding new commits to address the review comments or rebasing the PR & editing the commits to ensure each commit reflects the latest state of the code?

Reply:
thanks. just add new commits and push the branch. our preference is to follow the development and have individual commits recorded as changes happen (and information why) instead of having a "clean" commit history. once a branch is public any rebasing will lead to diverging commit histories and make small modifications to a pull request from the maintainers more complicated.